pkg/lib/cockpit.js

```diff
@@ -4212,9 +4212,9 @@
 
             return self.request({
                 method: "POST",
-                path: path,
-                body: body,
-                headers: headers
+                path,
+                body,
+                headers
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)